eth/eventdatahandler/handlers.go

```diff
@@ -0,0 +1,560 @@
+package eventdatahandler
```

Comment:
@pavelkrolevets / @nkryuchkov 
TODO: might be too big to be part of this PR, we need to add tests to each of the handlers , should probably be a mock DB where we test that after running the handler we check that the DB state is as expected.

Reply:
added a TODO